spec/spec_helper.rb

```diff
@@ -19,6 +19,7 @@
 
   config.before :all do
     FactoryBot.reload
+    Settings.first_or_create!(scores_locked: false)
```

Comment:
Probably better to use `instance`. In particular, this won't work if someone already has a `Settings` with locked scores in their test database (like I do from previously running the specs).

```suggestion
    Settings.instance.update!(scores_locked: false)
```

Reply:
Oh, and this should go in the `rails_helper.rb` (since it technically requires the Rails environment).